src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -560,9 +561,15 @@ export function handleGeoPushpinDrillEvent(
     drillConfig: IDrillConfig,
     execution: Execution.IExecutionResponses,
     geoData: IGeoData,
-    properties: GeoJSON.GeoJsonProperties,
+    feature: IGeoDataSourceFeature,
```

Comment:
Just small suggestion is that we should pass `properties` as `pinProperties: GeoJSON.GeoJsonProperties` and `pinCoordinator: string[]` as params to `handleGeoPushpinDrillEvent`, because `feature` is quite big and general IMO

Reply:
updated